site-permissions/site-permissions-store/src/main/java/com/duckduckgo/site/permissions/store/sitepermissions/SitePermissionsEntity.kt

```diff
@@ -25,6 +25,7 @@ data class SitePermissionsEntity(
     @PrimaryKey val domain: String,
     val askCameraSetting: String = ASK_EVERY_TIME.name,
     val askMicSetting: String = ASK_EVERY_TIME.name,
+    val askDrmSetting: String = ASK_EVERY_TIME.name,
```

Comment:
Changing an entity requires a DB migration otherwise will crash when updating the app and using this feature. In `SitePermissionsDatabase` you need to change the DB version from `1` to `2`.  It will also require to properly migrate the data to add the new column with ASK_EVERY_TIME as the default value.

Reply:
You can apply this patch to solved this.

[Add_db_migration.patch](https://github.com/duckduckgo/Android/files/13188182/Add_db_migration.patch)

Reply:
Great catch and thanks a lot for the patch! Applied and added steps to test DB migration.